packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -327,7 +327,7 @@ module.exports = new Transformer({
                 highlightParent = parent;
                 highlightedNodes = [];
                 return [];
-              } else if (node.properties?.className === 'comment' && /- end highlight -/.test(node.children[0].value)) {
+              } else if (node.properties?.className === 'comment' && /- end highlight -/.test(node.children?.[0].value)) {
```

Comment:
Some recent changes from main cause the docs-verdaccio build step to fail, so this fixes that.